lib/internal/process/next_tick.js

```diff
@@ -6,12 +6,10 @@
 // for details).
 const kMaxCallbacksPerLoop = 1e4;
 
-exports.setup = setupNextTick;
-
 function setupNextTick() {
   const promises = require('internal/process/promises');
   const errors = require('internal/errors');
-  const emitPendingUnhandledRejections = promises.setup(scheduleMicrotasks);
+  const emitPendingUnhandledRejections = promises(scheduleMicrotasks);
```

Comment:
I know the other 'setup' instances had the benefit of not having to be assigned to a variable first, but perhaps we should change the name here to something more straight-forward at least, like `setupPromises` instead of just `promises`?